vue-components/src/styles/mixins/Label.scss

```
@@ -1,8 +1,12 @@
-@mixin Label {
+@mixin Label( $displayType: 'inline' ) {
     color: $wikit-Label-font-color;
-    padding-block-end: $wikit-Label-padding-block-end;
     font-family: $wikit-Label-font-family;
     font-size: $wikit-Label-font-size;
     font-weight: $wikit-Label-font-weight;
     line-height: $wikit-Label-line-height;
-  }
\ No newline at end of file
+
+    @if $displayType != 'inline' {
+        display: $displayType;
+        padding-block-end: $wikit-Label-padding-block-end;
+    }
```

I think in this case rather than performing a negative check, I would prefer the display property to be fully controlled by the call and only the relevant styles applied on a positive basis rather than on negative (i.e. only on blaock rather than, "anything that is not inline"):

```suggestion
     display: $displayType;
     
    @if $displayType == block {
        padding-block-end: $wikit-Label-padding-block-end;
    }
```